community/sway/etc/skel/.config/foot/foot.ini

```
@@ -0,0 +1,156 @@
+# -*- conf -*-
+
+# shell=$SHELL (if set, otherwise user's default shell from /etc/passwd)
+# term=foot (or xterm-256color if built with -Dterminfo=disabled)
+# login-shell=no
+
+font=RobotoMono Nerd Font Mono:size=8
+# font-bold=<bold variant of regular font>
+# font-italic=<italic variant of regular font>
+# font-bold-italic=<bold+italic variant of regular font>
+# line-height=<font metrics>
+# letter-spacing=0
+# horizontal-letter-offset=0
+# vertical-letter-offset=0
+# underline-offset=<font metrics>
+# box-drawings-uses-font-glyphs=no
+# dpi-aware=yes
+
+# initial-window-size-pixels=700x500  # Or,
+# initial-window-size-chars=<COLSxROWS>
+# initial-window-mode=windowed
+# pad=2x2                             # optionally append 'center'
+resize-delay-ms=50
+
+# notify=notify-send -a ${app-id} -i ${app-id} ${title} ${body}
+
+# bold-text-in-bright=no
+# bell=none
+# word-delimiters=,│`|:"'()[]{}<>
+# selection-target=primary
+# workers=<number of logical CPUs>
+
+[bell]
+# urgent=no
+# notify=no
+# command=
+# command-focused=no
+
+[scrollback]
+lines=10000
+# multiplier=3.0
+# indicator-position=relative
+# indicator-format=
+
+[url]
+# launch=xdg-open ${url}
+# label-letters=sadfjklewcmpgh
+# osc8-underline=url-mode
+# protocols = http, https, ftp, ftps, file, gemini, gopher
+
+[cursor]
+# style=block
+color= 141a1b eeeeee
+# blink=no
+# beam-thickness=1.5
+# underline-thickness=<font underline thickness>
+
+[mouse]
+# hide-when-typing=no
+# alternate-scroll-mode=yes
+
+[colors]
+alpha=0.9
+foreground=eeeeee
+background=141a1b
+regular0=141a1B  # black
+regular1=cd3f45  # red
+regular2=9fca56  # green
+regular3=e6cd69  # yellow
+regular4=16a085  # blue
+regular5=a074c4  # magenta
+regular6=55b5db  # cyan
+regular7=d6d6d6  # white
+bright0=41535B   # bright black
+bright1=cd3f45  # red
+bright2=9fca56  # green
+bright3=e6cd69  # yellow
+bright4=16a085  # blue
+bright5=a074c4  # magenta
+bright6=55b5db  # cyan
+bright7=ffffff   # bright white
+# 16 = <256-color palette #16>
+# ...
+# 255 = <256-color palette #255>
+# selection-foreground=<inverse foreground/background>
+# selection-background=<inverse foreground/background>
+# jump-labels=<regular0> <regular3>
+# urls=<regular3>
+
+[csd]
+# preferred=server
```

shouldn't we disable csd?
```suggestion
preferred=none
```

---

Hm, I don't see any decorations even with the default config, probably because gtk3-nocsd is the default gtk3 provider for Manjaro Sway. If it affects anything else then sure, why not!

---

`preferred` is just that - we ask the compositor very nicely to use either SSD (`preferred=server`), or CSD (`preferred=client). The compositor may choose to override that. Sway for example, does.

Furthermore, this setting is completely ignored on compositors that doesn't implement the _XDG toplevel decoration_ Wayland protocol (only GNOME/mutter, that I'm aware of).

`preferred=none` on the other hand will disable **both** SSD and CSDs (assuming a floating window in Sway - tiled and stacked windows get SSDs anyway).

---

Ah, thx for clarification!